app/main/services/search_service.py

```diff
@@ -157,7 +157,7 @@ def url_for_search(**kwargs):
     except TransportError as e:
         try:
             root_causes = getattr(e, "info", {}).get("error", {}).get("root_cause", {})
-        except AttributeError as e:
+        except AttributeError:
```

Comment:
How does this exception ever get raised if we're setting default return values of dicts?

Reply:
if `getattr(e, "info", {})` returns a string, you'll get an attribute error

Reply:
yep ofc